sdk/confidentialledger/Azure.ResourceManager.ConfidentialLedger/src/autorest.md

```diff
@@ -9,10 +9,11 @@ generate-model-factory: false
 csharp: true
 library-name: ConfidentialLedger
 namespace: Azure.ResourceManager.ConfidentialLedger
-require: https://github.com/Azure/azure-rest-api-specs/blob/e7bcafa885ef773c6309d6a8f3a65c5019df413d/specification/confidentialledger/resource-manager/readme.md
+require: https://github.com/Azure/azure-rest-api-specs/blob/756495dd7e0e2f5181039def47a8c85ff0787b66/specification/confidentialledger/resource-manager/readme.md
 output-folder: $(this-folder)/Generated
 clear-output-folder: true
 skip-csproj: true
+tag: package-preview-2023-01
```

Comment:
remove the tag config if it is the default one in readme.md

Reply:
Removed the tag. Thanks!